config/seatunnel.yaml

```diff
@@ -37,3 +37,7 @@ seatunnel:
     telemetry:
       metric:
         enabled: false
+      log:
+        scheduled-deletion-enable: false
+        cron: "0 0 * * *"
```

Comment:
I think we should check if this property is valid at cluster startup. Because Cron cron = parser.parse(cronExpression) this may throw IllegalArgumentException.

Reply:
get，I will make changes